policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+### The default development branch
+
+- Any type (bug fix, feature, refactoring, ...) of pull requests is allowed.
+- The development of the next minor or major release happens there.
+  API/ABI breaking changes are allowed on this branch only if the next
+  release will be a major release.
+- Any changes merged to this branch must be ported to the future major
+  and future minor branches if they are applicable. This can be done by
+  directly cherry-picking the changes when merging if there are no conflicts.
+  By this we must ensure that no features or bug fixes are unintentionally
+  lost in future major or minor releases.
+
+### The supported release branches
+
+- The development of the next patch releases of supported stable releases
+  happens there.
+- According to [stable release update policy] only bug fixes and
+  documentation changes are allowed.
+- By exception given by OMC also other types of pull requests can be merged.
+- Bug fix and documentation pull requests must be always merged to the
+  latest branch where the bug or documentation deficiency is present including
+  the future major and minor branches.
+  It can be then merged (backported or directly cherry-picked) to all
+  older branches where the deficiency is present.
```

Comment:
If someone reports an error in 1.1.1, I usually make the fix against 1.1.1, not master.  Does this section contradict that?  I'm a little unsure how to interpret the wordings for that kind of scenario.

Reply:
I'll wait for the answer before voting

Reply:
IMO it does not preclude that, if you ensure that the fix (if the bug applies to 3.0 and master) applies to later branches. If it does not apply cleanly you'll need to create the PR against these branches as well. This talks about merging, i.e. no fix should be merged to 1.1.1 (if the bug is on 3.0 as well) without merging also to 3.0 and master first (or at least immediately after).

Reply:
> This talks about merging, i.e. no fix should be merged to 1.1.1 (if the bug is on 3.0 as well) without merging also to 3.0 and master first (or at least immediately after).

Agreed, and that's how I usually do things